CHANGELOG.md

```diff
@@ -5,8 +5,12 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
-
 ### Added
+- `conjurctl server` and `conjurctl account create` allow the operator to specify the admin user's
+  password via STDIN by providing the `--password-from-stdin` switch.
+  [cyberark/conjur#2043](https://github.com/cyberark/conjur/issues/2043)]
+- `conjurctl account create` now allows the operator to specify the account name via the `--name` flag.
```

Comment:
previously this was `conjurctl account create {account}` - what is the benefit of adding a flag? we might want to add a note about that here

Reply:
Without adding a `--name` flag to `conjurctl account create`
`conjurctl account create --password-from-stdin <account_name>` is valid
`conjurctl account create <account_name>  --password-from-stdin` is invalid

To ensure the user is able to provide the `--password-from-stdin` and the `account name` independent of order. I decided to add the `name flag`. I will add a comment highlighting this in the CHANGELOG.
Thanks for the feedback!